.github/workflows/moodle-ci.yml

```diff
@@ -31,20 +31,26 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: ['7.3', '7.4']
-        moodle-branch: ['MOODLE_310_STABLE', 'MOODLE_311_STABLE']
+        php: ['8.0']
+        moodle-branch: ['MOODLE_401_STABLE', 'MOODLE_402_STABLE', 'MOODLE_403_STABLE']
         database: [pgsql, mariadb]
         include:
```

Comment:
Why are we testing different Moodle branches with both Postgres and MariaDB? Is that recommended that all integrations is expected to work with both DBs?

Reply:
Moodle versions expect specific versions of PHP to work. We have to define the matrix like this to ensure the  Moodle versions match their PHP versions. Ideally, we ought to test the last 3 major versions.

Reply:
We don't have control over which version and database our clients are using hence it is recommended to use multiple setups in the CI tests.